config/analytics_blocklist.yml

```diff
@@ -420,3 +420,13 @@
   - created_at
   - updated_at
   - provider_id
+  :bulk_update_placement_rows:
```

Comment:
Do we need to let data insight team know?

Reply:
this is in the blocklist so I don't think so?